src/platforms/common/data-management/sensitive-data/index.mdx

```diff
@@ -59,8 +59,13 @@ There's a few areas you should consider that sensitive data may appear:
 - Breadcrumbs → some SDKs (for example, JavaScript, Java logging integrations) will pick up previously executed log statements. **Do not log PII** if using this feature and including log statements as breadcrumbs in the event. Some backend SDKs will surface DB queries which may need to be scrubbed
 - User context → automated behavior is controlled via `send-default-pii`
 - HTTP context → query strings may be picked up in some frameworks as part of the HTTP request context
+- Transaction Names → In certain situations, transaction names might contain sensitive data.
+  For example, a browser's pageload transaction might have the raw URL as a name (e.g. `/users/1234/details`, where `1234` is the user id).
+  Our SDKs manage to parameterize URLs and routes in most cases successfully (i.e. turn `/users/1234/details` into `/users/:userid/details`).
```

Comment:
(This is Isabel's exact suggestion, but with a comma instead of a colon.)

```suggestion
In most cases, our SDKs can parameterize URLs and routes successfully, that is, turn `/users/1234/details` into `/users/:userid/details`.
```